gamemodes/cinema/gamemode/modules/theater/services/sh_file.lua

```diff
@@ -62,6 +62,9 @@ if CLIENT then
 
 					if duration>0 then
 						callback({duration=duration})
+					elseif duration<2147483647 then
```

Comment:
this if/elseif/else block doesn't make sense, what values do you expect for these cases?

Reply:
Hacked/corrupted files return large durations so if the duration is over the 32 bit limit then it's probably a hacked file.

Reply:
> Yeah but my fix is like 3 statements in one file
mine is a single if statement lol

Reply:
you're using < not >